.github/actions/before-install/action.yml

```diff
@@ -0,0 +1,49 @@
+name: "Before install and variables setup"
+description: "Before install and variables setup"
+
+inputs:
+  artifact:
+    description: 'path to the artifact to archieve (tar.bz2) and upload (like ./dist)'
+    required: true
+    type: string
+  output:
+    description: 'the S3 object to copy it to, like: s3://bucket-name/folder/whatever.tar.bz2'
+    required: true
+    type: string
+
+runs:
+  using: "composite"
+  steps:
+    - uses: Alfresco/alfresco-build-tools/.github/actions/get-commit-message@v1.35.0
+    - name: setup variables
+      shell: bash
+      run: |
+        echo "BUILD_OPTS=--configuration=production,e2e" >> $GITHUB_ENV
+        echo "TEST_OPTS=" >> $GITHUB_ENV
+        echo "E2E_PROTRACTOR_OPTS=" >> $GITHUB_ENV
+        echo "E2E_TSCONFIG=tsconfig.e2e.json" >> $GITHUB_ENV
+        echo "GIT_HASH=$(git rev-parse HEAD)" >> $GITHUB_ENV
+        echo "SMART_RUNNER_DIRECTORY=.protractor-smartrunner" >> $GITHUB_ENV
+        echo "BASE_HASH=.protractor-smartrunner" >> $GITHUB_ENV
+        echo "HEAD_HASH=HEAD" >> $GITHUB_ENV
+    - name: setup S3 caching
+      shell: bash
+      run: |
+        # pip install -q awscli
+        S3_DBP_PATH="s3://alfresco-travis-builds/aca"
```

Comment:
should we remove the word "travis" from GHA actions where possible?

Reply:
@gionn can we just change s3 path? Or is some config on s3 side should be done beforehand? I think I tried changing this but I got an error